autogpt_platform/backend/backend/blocks/io.py

```diff
@@ -193,6 +197,7 @@ async def run(self, input_data: Input, *args, **kwargs) -> BlockOutput:
         """
         if input_data.format:
             try:
+                formatter = TextFormatter(autoescape=input_data.escape_html)
```

Comment:
Can we make sure that this is safe?

Reply:
- It still uses the `SandboxedEnvironment`, which the docs state can be used to render untrusted templates
- It effectively does the same as `{{ value | safe }}` on all inserted values